data/interfaces/default/info.html

```diff
@@ -894,6 +947,28 @@ <h4 class="modal-title">Recently Added Notification</h4>
     });
 </script>
 % endif
+% if data['media_type'] in ('show', 'movie'):
```

Comment:
`% if data['media_type'] in ('movie', 'show', 'season', 'episode', 'artist', 'album', 'track'):`